src/wgquickprocess.cpp

```diff
@@ -12,40 +12,41 @@
 namespace {
 Logger logger(
 #if defined(MVPN_LINUX)
-    LOG_LINUX
+  LOG_LINUX
 #elif defined(MVPN_MACOS_DAEMON)
-    LOG_MACOS
+  LOG_MACOS
 #endif
-    ,
-    "WgQuickProcess");
+  ,
+  "WgQuickProcess");
 
 QString scriptPath() {
-#if defined(MVPN_LINUX)
-  return "wg-quick";
-#elif defined(MVPN_MACOS_DAEMON)
   QDir appPath(QCoreApplication::applicationDirPath());
   appPath.cdUp();
   appPath.cd("Resources");
   appPath.cd("utils");
   return appPath.filePath("helper.sh");
-#else
-#  error Unsupported platform
-#endif
 }
 
 }  // namespace
 
-// static
-bool WgQuickProcess::run(
-    Daemon::Op op, const QString& privateKey, const QString& deviceIpv4Address,
+QString const WgQuickProcess::FAIL = "FAIL";
```

Comment:
In general, as a convention, you can use an empty string too.
In this way, the caller of writeWgConfigFile can do:

```
QString fileName = WgQuickProcess::writeConfigFile(...);
if (fileName.isEmpty()) {
  logger.log() << "Something went wrong";
}
```

but this the FAIL const approach works too!